src/data/location_item_menu_layout.ts

```diff
@@ -451,7 +453,9 @@ export const shop_item_menu_layout = [
     ],
     [
         'Buy Blue Fire',
+        'Buy Fairy\'s Spirit',
         'Buy Deku Stick (1)',
+        'Buy Deku Nuts (10)',
```

Comment:
This should be `Buy Deku Nut (10)` to match rando's item list. Also needs to be changed in the vertical layout.